index.d.ts

```diff
@@ -302,24 +358,30 @@ declare namespace Moleculer {
 		serializer?: Serializer;
 		validator?: Validator;
 		transit: GenericObject;
+		middlewares: GenericObject;
```

Comment:
Ok. It is a new class named `MiddlewareHandler`: https://github.com/moleculerjs/moleculer/blob/master/src/middleware.js
It has some methods, but maybe no need to publish it because it is an internal class, maybe will be changed in the future.

Reply:
Is the `MiddlewareHandler` class an internally created only object? If so I'll make it an interface as it isn't to have any instances created outside the internal mechanism of moleculer.

Reply:
yes, it's internally created & handled.

Reply:
Sorry missed that in the previous comment, so is there currently any reason for someone to access this property when creating a service?

Reply:
Maybe. For example, if someone wants to create a new custom `Service` class because `Service` [should call it for action handlers to wrap with middlewares](https://github.com/moleculerjs/moleculer/blob/master/src/service.js#L95)